xsuite/src/cli/newWalletCmd.ts

```diff
@@ -37,19 +41,29 @@ const action = async ({
       return;
     }
   } else {
-    keystore = Keystore.createFile_unsafe(walletPath, password);
+    let data: any;
```

Comment:
`data` can be defined inside the else and be a const.

But a simpler way would be to move `keystore = Keystore.createFile_unsafe(walletPath, password, data);` out of the if/else.

By default, data is set undefined. If the pem file is defined, then the data is set.

Reply:
👍

Reply:
Doesn't seem fixed

Reply:
Doesn't seem fixed

Reply:
I see it is addressed in the next PR, so all good